package.json

```diff
@@ -30,10 +30,7 @@
   },
   "peerDependencies": {
     "babel-eslint": "^6.0.4",
-    "eslint": "^2.7.0",
-    "eslint-plugin-ember": "^1.1.0",
-    "eslint-plugin-ember-cleanup": "^1.6.0",
-    "eslint-plugin-lodash": "^1.8.2"
+    "eslint": "^2.7.0"
```

Comment:
Yeah, I think they make sense.  The problem with the plugin ones is they only conditionally make sense.